consensus/clique/snapshot.go

```diff
@@ -315,7 +308,9 @@ func (s *Snapshot) signers() []common.Address {
 	for sig := range s.Signers {
 		sigs = append(sigs, sig)
 	}
-	sort.Sort(signersAscending(sigs))
+	slices.SortFunc(sigs, func(a, b common.Address) bool {
```

Comment:
I think you could reasonably define a `Less` function on `common.Address`. I decided not to do so in this PR, but something to consider for the maintainers.